src/components/switch/switch.component.ts

```diff
@@ -179,46 +186,69 @@ export default class SlSwitch extends ShoelaceElement implements ShoelaceFormCon
   }
 
   render() {
+    const hasHelpTextSlot = this.hasSlotController.test('help-text');
+    const hasHelpText = this.helpText ? true : !!hasHelpTextSlot;
```

Comment:
```suggestion
    const hasHelpText = this.helpText || hasHelpTextSlot;
```

^ I think that should work. `.test` returns a boolean

Reply:
I actually copied this line from [Input](https://github.com/shoelace-style/shoelace/blob/next/src/components/input/input.component.ts#L410). Want me to change Input and other instances of this line as well?

Reply:
@clintcs 😅 then just leaving it as is is fine!